Assignment/Complex.h

```diff
@@ -0,0 +1,31 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Vinitha Perumal - GET
+// ------------------------------------------------------------------
+// Complex.h
+// Program on Test1.2 branch.
+// ------------------------------------------------------------------------------------------------
+#ifndef COMPLEX_H
+#define COMPLEX_H
+
+typedef struct ComplexNumber {
+   float real, img;
+}ComplexNumber;
+
+/// <summary>To Add two complex numbers.</summary>
+ComplexNumber Op_Add (ComplexNumber num1, ComplexNumber num2);
+
+/// <summary>To Subtract one complex number from another.</summary>

```

Comment:
```suggestion
/// <summary>Returns the difference of 2 complex numbers</summary>

```
The comments over a function describe what the function returns, and what parameters it requires (if it is not already obvious)